backstop_data/engine_scripts/puppet/clickAndHoverHelper.js

```diff
@@ -0,0 +1,23 @@
+module.exports = async (page, scenario) => {
+  var hoverSelector = scenario.hoverSelectors || scenario.hoverSelector;
+  var clickSelector = scenario.clickSelectors || scenario.clickSelector;
+  var postInteractionWait = scenario.postInteractionWait; // selector [str] | ms [int]
+
+  if (hoverSelector) {
+    for (const hoverSelectorIndex of [].concat(hoverSelector)) {
+      await page.waitFor(hoverSelectorIndex);
+      await page.hover(hoverSelectorIndex);
+    }
+  }
+
+  if (clickSelector) {
+    for (const clickSelectorIndex of [].concat(clickSelector)) {
+      await page.waitFor(clickSelectorIndex);
+      await page.click(clickSelectorIndex);
+    }
+  }
+
```

Comment:
the helper is called `clickAndHover` but it seems it first hover and then clicks?

Reply:
backstop stuff